crowd_anki/github/github_importer.py

```diff
@@ -8,6 +8,16 @@
 
 BRANCH_NAME = "master"
 
+
+def get_repository_name(repository_url):
+    repo_url_path = urlparse(repository_url).path
+    repo_name = list(filter(None, repo_url_path.split("/")))[-1]  # using filter() in case it ends in "/"
+    repo_name = repo_name.split(".")[0]  # in case it ends in ".git"
+    if repo_name.find(" ") != -1:
+        raise ValueError("repository name in URL should not contain spaces")
```

Comment:
that's seems like an oddly specific thing to check for. should we just strip the string?

Reply:
Took it out as well. I think the reason I was trying to validate the input so much was that Dulwich does create a folder and empty repository even when the URL is not correct, or does not contain a git repository. But we can handle that in other ways, perhaps.